mlir/test/Conversion/AffineToStandard/lower-affine.mlir

```diff
@@ -657,3 +657,21 @@ func @affine_max(%arg0: index, %arg1: index) -> index{
   %0 = affine.max affine_map<(d0,d1) -> (d0 - d1, d1 - d0)>(%arg0, %arg1)
   return %0 : index
 }
+
+// CHECK-LABEL: func @dot(
+// CHECK-SAME: %[[ARG0:.*]]: memref<100x100xf32>, %[[ARG1:.*]]: memref<100x100xf32>) {
+func @dot(%o: memref<100x100xf32>, %a: memref<100x100xf32>) {
+  affine.parallel (%i, %j) = (0, 0) to (100, 100) {
+    %0 = affine.load %a[%i, %j] : memref<100x100xf32>
+  }
+  return
+}
+
+// CHECK:    %[[Cm1:.*]] = constant 100 : index
+// CHECK:    %[[Cm2:.*]] = constant 100 : index
+// CHECK:    %[[Dm1:.*]] = constant 0 : index
+// CHECK:    %[[Dm2:.*]] = constant 0 : index
+// CHECK:    %[[Em1:.*]] = constant 1 : index
+// CHECK:    %[[Em2:.*]] = constant 1 : index
```

Comment:
Use CHECK-DAG instead of CHECK here.

Reply:
No need of matching all the way until `: index` - you can drop the part `: index`.